extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -215,6 +221,17 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
      * @private {?../../amp-story/0.1/amp-story-store-service.AmpStoryStoreService|?../../amp-story/1.0/amp-story-store-service.AmpStoryStoreService}
      */
     this.storeService_ = null;
+
+    /** @private @const {!LocalizationService} */
+    this.localizationService_ = new LocalizationService(this.win);
```

Comment:
Should we have a service getter (eg: `Services.localizationServiceFor(this.win)`) that could be shared by multiple extensions instead?

Reply:
We can; let me send in a followup refactoring of localization more generally.  I'll file an issue, as there are a number of things that are not ideal.